x-pack/plugins/security_solution/public/overview/components/detection_response/host_alerts_table/host_alerts_table.tsx

```diff
@@ -0,0 +1,172 @@
+  navigateTo: NavigateTo;
+}) => Array<EuiBasicTableColumn<AlertSeverityCounts>>;
+
+const DETECTION_RESPONSE_HOST_SEVERITY_QUERY_ID = 'vulnerableHostsBySeverityQuery';
+
+export const HostAlertsTable = React.memo(() => {
+  const { getAppUrl, navigateTo } = useNavigation();
+  const { toggleStatus, setToggleStatus } = useQueryToggle(
+    DETECTION_RESPONSE_HOST_SEVERITY_QUERY_ID
+  );
+
+  const { data, isLoading } = useHostAlertsItems({
+    skip: !toggleStatus,
+    queryId: DETECTION_RESPONSE_HOST_SEVERITY_QUERY_ID,
+  });
+
+  const columns = useMemo(
+    () => getTableColumns({ getAppUrl, navigateTo }),
+    [getAppUrl, navigateTo]
+  );
+
+  return (
+    <EuiPanel hasBorder data-test-subj="hostSeverityAlertsPanel">
+      <HeaderSection
+        id={DETECTION_RESPONSE_HOST_SEVERITY_QUERY_ID}
+        title={i18n.HOST_ALERTS_SECTION_TITLE}
+        titleSize="s"
+        hideSubtitle
+        toggleStatus={toggleStatus}
+        toggleQuery={setToggleStatus}
+      />
+      {toggleStatus &&
+        (isLoading || data.length > 0 ? (
+          <>
+            <EuiBasicTable
+              data-test-subj="hostAlertsTable"
+              columns={columns}
+              items={data}
+              loading={isLoading}
+            />
+            <EuiSpacer size="m" />
+            <EuiButton
+              onClick={() => navigateTo({ url: getAppUrl({ deepLinkId: SecurityPageName.hosts }) })}
+            >
+              {i18n.VIEW_ALL_HOST_ALERTS}
+            </EuiButton>
+          </>
+        ) : (
+          <EuiEmptyPrompt title={<h3>{i18n.NO_ALERTS_FOUND}</h3>} titleSize="xs" />
+        ))}
+    </EuiPanel>
+  );
+});
+
+HostAlertsTable.displayName = 'HostAlertsTable';
+
+const getTableColumns: GetTableColumns = ({ getAppUrl, navigateTo }) => [
+  {
+    field: 'hostName',
+    name: i18n.HOST_ALERTS_HOSTNAME_COLUMN,
+    truncateText: true,
+    textOnly: true,
+    render: (hostName: string) => {
+      const url = getAppUrl({ deepLinkId: SecurityPageName.hosts, path: hostName });
+
+      return (
+        <EuiLink
+          data-test-subj="userSeverityAlertsTable-hostName"
+          href={`/app/security/hosts/${hostName}`}
+          onClick={(ev?: React.MouseEvent) => {
+            if (ev) {
+              ev.preventDefault();
+            }
+            navigateTo({ url });
+          }}
+        >
+          {hostName}
+        </EuiLink>
+      );
+    },
+  },
+  {
+    field: 'totalAlerts',
+    name: i18n.ALERTS_COLUMN,
+    render: (totalAlerts: number, { hostName }) => {
+      const url = getAppUrl({ deepLinkId: SecurityPageName.hosts, path: hostName });
+
+      return (
+        <EuiLink
+          data-test-subj="hostSeverityAlertsTable-totalAlerts"
+          href={`/app/security/hosts/${hostName}`}
+          onClick={(ev?: React.MouseEvent) => {
+            if (ev) {
+              ev.preventDefault();
+            }
+            navigateTo({ url });
+          }}
+        >
+          {totalAlerts}
```

Comment:
Do we need to format the number with `,` here?  like x-pack/plugins/security_solution/public/network/components/network_top_n_flow_table/columns.tsx Line 217-220?

Reply:
Good catch, I agree we should format all counters. However, the `0,000` format is the English standard, but it is [not the same everywhere](https://docs.oracle.com/cd/E19455-01/806-0169/overview-9/index.html). 
We could do `Number(totalAlerts).toLocaleString();`, so it will take the locale from the browser. 
But I think the best option will be to use the same `FormattedCount` (1K, 3M,...) that we already have in the alerts charts. Anyway, it would be good to confirm that with Paul or the UX team.

BTW, I also added the "alerts count" column without format in the rules table. Whatever criteria you apply, could you please do the same there? @angorayc

Reply:
<img width="597" alt="Screenshot 2022-04-15 at 01 44 27" src="https://user-images.githubusercontent.com/6295984/163445438-44a3e96c-9cde-40e0-9739-92953fd6e18b.png">

Reply:
really great catch, I never really think about number formatting or even the persons locale. Thanks, Ive messaged Paul as well will update the slack with whatever format he decides, and for now ill go with Formatted count